release/tigervnc.iss.in

```diff
@@ -24,10 +24,40 @@ LicenseFile=@CMAKE_SOURCE_DIR@\LICENCE.txt
 Name: "{sys}\config\systemprofile\Desktop"
 
 [Files]
+#ifndef BUILD_STATIC
+Source: "@WIN_DLL_PATH@libintl-8.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libiconv-2.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@mgwfltknox-1.3.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@zlib1.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@libjpeg-8.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@libgcc_s_seh-1.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@libstdc++-6.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@libwinpthread-1.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+#ifdef ENABLE_GNUTLS
+Source: "@WIN_DLL_PATH@libffi-7.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libgmp-10.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libgnutls-30.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libhogweed-6.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;  
+Source: "@WIN_DLL_PATH@libidn2-0.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libnettle-8.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libp11-kit-0.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libtasn1-6.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libunistring-2.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+#endif
+#else
+Source: "@WIN_DLL_PATH@libffi-7.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@libp11-kit-0.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libunistring-2.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@libintl-8.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libiconv-2.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+#endif
```

Comment:
I'm afraid I don't think this is something we can really maintain. The list of DLLs and versions will vary depending on the build environment so this list will never be correct. :/

Reply:
Afraid not. I'm not familiar with how open source projects usually deal with Windows packaging. Should be possible to find some examples though?

Reply:
I will see what I can find.  It probably won't be quick.

Reply:
I am removing this section from my pull request for now.